fs/expose/dokan/__init__.py

```diff
@@ -76,7 +85,9 @@
 from fs.wrapfs import WrapFS
 
 try:
-    import libdokan
+    if six.PY2: import libdokan
+    elif six.PY3: from . import libdokan
```

Comment:
So, why `elif six.PY3` and what is the `else` for? From what I see this code unnecessarily special-cases Python 3. ``PY3 = sys.version_info[0] == 3`, so this code will break on Python 4 for no reason.